examples/config.mjs

```diff
@@ -0,0 +1,50 @@
+// config.mjs
+import * as dotenv from "dotenv";
+import * as fs from "fs";
+import * as path from "path";
+import { SayariError } from "@sayari/sdk";
+
+// Define potential paths for the .env file
+const currentPath = path.resolve(".env");
+const parentPath = path.resolve("../.env");
+
+// Load environment variables from the appropriate .env file
+if (fs.existsSync(currentPath)) {
+    dotenv.config({ path: currentPath });
+} else if (fs.existsSync(parentPath)) {
+    dotenv.config({ path: parentPath });
+} else {
+    console.warn("Warning: .env file not found in current or parent directory.");
```

Comment:
We may want to only trigger this 'if/else' logic if the ENV vars are not already set. I wouldn't want to trigger this warning if a user has elected to export the vars directly instead of using an env file (my apologies if that isn't possible here)

Reply:
Good spot! 👍